pulser-core/pulser/backend/remote.py

```diff
@@ -96,7 +99,12 @@ class RemoteConnection(ABC):
 
     @abstractmethod
     def submit(
-        self, sequence: Sequence, wait: bool = False, **kwargs: Any
+        self,
+        sequence: Sequence,
+        wait: bool = False,
+        complete: bool = False,
+        batch_id: str | None = None,
```

Comment:
Since we are renaming `submission_id` with `batch_id`, we should do it everywhere on the methods below too